gobblin-hive-registration/src/main/java/gobblin/hive/avro/HiveAvroSerDeManager.java

```diff
@@ -52,20 +55,26 @@
   public static final String DEFAULT_SCHEMA_FILE_NAME = "_schema.avsc";
   public static final String SCHEMA_LITERAL_LENGTH_LIMIT = "schema.literal.length.limit";
   public static final int DEFAULT_SCHEMA_LITERAL_LENGTH_LIMIT = 4000;
+  public static final String HIVE_SPEC_SCHEMA_READING_TIMER = "AvroReading";
```

Comment:
Please make this name more precise. For example: `hiveAvroSerdeManager.schemaReadTimer` (same for next line).